### PR TITLE
Cu 8yduxz software dev docs john carr

### PR DESCRIPTION
**Pull request by John-Carr**
I almost broke it again I am bad at this sorry

### CODE REVIEW

**Comment by John-Carr** [comment]
Task linked: [CU-8yduxz Software Dev Docs](https://app.clickup.com/t/8yduxz)

File: Processes/HardwareDevelopment.md [hunk 1]
```diff
@@ -4,13 +4,23 @@
...
+**To create a new project**
 1. Make sure you are a member of the [Solar-Gators GitHub organization](https://github.com/Solar-Gators).
 2. Go to the organization homepage.
 3. Towards the left of the screeen there is a green button with the text new, press that button.
 4. This will take you to the create a new repository page fill out the information requested following the above conventions.
 5. Make sure to add this new repo to clickup. There is a guide [here](https://docs.clickup.com/en/articles/856285-github).  
+**To Migrate an existing project**
+1. Follow the above steps to create a new repository and copy the url.
+2. On your local computer go to the directory where the PCB is and open git bash.
+3. Run `git clone your-repo-url .`.
```
Comment (John-Carr) [inline Processes/HardwareDevelopment.md:16]: I need to confirm this actually works with a directory that has stuff in it might have to do some jank stuff.

**Comment by Seth10001** [comment]
So it gets built to here: https://solar-gators-docs--5.org.readthedocs.build/en/5/. Not sure how to find this URL anywhere else

**Comment by Seth10001** [comment]
Actually, if you click the "Details" link under "Show all checks" then it links to there!

File: Processes/index.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Developing
+=========
+
+The electrical team is responsible for developing hardware and software for the car.
+Every team has their own repositories and how they like to do things but here we will discuss things that are common.
+.. toctree::
```
Comment (Seth10001) [inline Processes/index.rst:7]: Not sure what happened the toctree didn't generate links but instead thought it was a code block?

**Review by Seth10001 — CHANGES_REQUESTED** [review]
Fix the toctree issue

See here: https://solar-gators-docs--5.org.readthedocs.build/en/5/Processes/index.html

**Pull request merged**